python/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "pylance"
-version = "0.7.3"
+version = "0.7.4"
```

Comment:
This was due to 44e8e79 already pushed 0.7.4 from rust side. Just bring python in sync with rust. No meant to do a new release.